### PR TITLE
Nvme show error

### PR DESCRIPTION
**Pull request by ikegami-t**

### CODE REVIEW

File: nvme-print.c [hunk 1]
```diff
@@ -5462,8 +5462,10 @@ void nvme_show_error(const char *msg, ...)
 	va_list ap;
 	va_start(ap, msg);
 
-	if (argconfig_output_format_json(false))
-		return json_output_error(msg, ap);
+	if (argconfig_output_format_json(false)) {
+		json_output_error(msg, ap);
+		return va_end(ap);
```
Comment (igaw) [inline nvme-print.c:5470]: `return va_end()` for a void function doesn't really make sense.

Comment (ikegami-t) [reply nvme-print.c:5470]: Thansks for your review comments. To make sure let confirm below.
The function to return is defined as the void function as below so seems it correct but still should it be separated the function call and retrun as mentioned?
```c
void va_end(va_list ap);
```

Comment (igaw) [reply nvme-print.c:5470]: Yes, the idea is to use plain `return;` if the function is defined as 'void', so don't return void. 

Obviously, compilers are able to handle this but my eyes are trained that I always check if the return types match. A plain return says, we do not return anything. It's a readability thing in short.

Comment (ikegami-t) [reply nvme-print.c]: Thanks for your explanation. I see so just fixed the patch as mentioned.

**Pull request merged**